actionpack/lib/action_dispatch/system_testing/test_helpers/undef_methods.rb

```diff
@@ -14,7 +14,7 @@ module UndefMethods # :nodoc:
 
           def method_missing(method, *args, &block)
             if METHODS.include?(method)
-              raise NoMethodError
+              raise NoMethodError, "system tests cannot make direct requests via ##{method}; use #visit and #click_on or such capybara API. See also http://www.rubydoc.info/github/teamcapybara/capybara/master#The_DSL"
```

Comment:
Hey @yalab can you capitalize the start of the sentence and end it with a period? Like this:

> "System tests cannot make direct requests via ##{method}; use #visit and #click_on or such capybara API. See also http://www.rubydoc.info/github/teamcapybara/capybara/master#The_DSL."

Reply:
Also can you update the sentence to say:

> "System tests cannot make direct requests via ##{method}; use #visit and #click_on. See http://www.rubydoc.info/github/teamcapybara/capybara/master#The_DSL for more information."